src/dialog.scss

```diff
@@ -159,15 +159,15 @@ $button: #{$fd-namespace}-button;
     @include fd-reset();
 
     position: absolute;
-    right: 0;
-    bottom: -4px;
+    right: 1px;
```

Comment:
<img width="682" alt="Screen Shot 2021-10-06 at 3 07 51 PM" src="https://user-images.githubusercontent.com/39598672/136267352-a9019202-b153-41c9-b581-f84ad3132255.png">

Please use mixin for setting absolute position, it will handle RTL and you won't need to put the value in a variable.